### DISCUSSION TITLE
NeoVim 0.10

### DISCUSSION THREAD

**Discussion by MunsMan** [mikesmithgh/kitty-scrollback.nvim#55] [Q&A] [answered]
I wondered if Neovim 0.10 is really required.
I'm currently running the last release, and I would love to avoid any nightly releases for now.
Likewise, I'm just curious, which features are you using from the nightly release?

**Comment by jackielii** [comment]
Wondering the same thing, was going to create an issue, good that I clicked discussion first.

I scrolled through commit history, but couldn't find anything related. So it must be an lua API that's only available in the nightly version?

@mikesmithgh looking forward to hearing from you.

**Comment by mikesmithgh** [comment]
Hey @MunsMan and @jackielii, thank you for the question.

I answered it in a similar discussion here: https://github.com/mikesmithgh/kitty-scrollback.nvim/discussions/53

I am using `vim.system` which is only in `v0.10`.

It seems like there is interested around this, so I may make it compatible with `v0.9` depending on my availability vs when `v0.10` is released as stable.

**Reply by jackielii** [reply]
> when v0.10 is released as stable.

Indeed, if v0.10 can be available shortly, that wouldn't affect too much of the adoption. Currently I'm hesitate to upgrade as last time I installed with brew --head, a lot of dependencies got broken. It does seems like a relatively small change

On the other hand, I can understand you don't want to spend time on it. I'm going to try nightly first see if I can get things running . If not, maybe I'll create a PR for this?

**Comment by mikesmithgh** [comment]
I created the issue https://github.com/mikesmithgh/kitty-scrollback.nvim/issues/58 to track this. Please upvote or comment on that issue so I can gauge interest. 👍

**Comment by mikesmithgh** [comment, marked as answer]
@MunsMan @jackielii kitty-scrollback.nvim now supports neovim `v0.9` 😸
